libsignal-protocol/Cargo.toml

```diff
@@ -18,6 +18,7 @@ sha2 = { version = "0.8.0", optional = true }
 hmac = { version = "0.7.0", optional = true }
 
 [features]
-default = ["crypto-native"]
+default = ["crypto-native", "in-memory-store"]
 crypto-native = ["sha2", "hmac"] # TODO(shekohex): find a way to add `aes`.
-crypto-openssl = ["openssl"]
\ No newline at end of file
+crypto-openssl = ["openssl"]
+in-memory-store = []
```

Comment:
i added that as a feature flag since not all people like to use the In Memory Store in Production :)